backend/skip_list.js

```diff
@@ -114,7 +114,7 @@ class SkipList {
   constructor (randomSource) {
     const head = new Node(null, null, 1, [], [null], [], [null])
     const random = randomSource ? randomSource() : randomLevel()
-    return makeInstance(0, Map().set(null, head), random)
+    return makeInstance(0, Map().set(null, head), random) // eslint-disable-line
```

Comment:
I'm not sure the best way to fix this, but generally speaking a Javascript constructor doesn't return the object created but manipulates "this". I suspect a fix will be a little bit tricky.